src/gameLink.js

```diff
@@ -1,4 +1,4 @@
-var crypto = require('crypto');
+//var crypto = require('crypto'); // NOTE: This seems to no longer be needed
```

Comment:
Used on line 45

Reply:
@dustmop Dependency is used but is now built in and does not need a require statement